tests/test_keys.py

```diff
@@ -682,6 +683,13 @@ def test_extract_pem(self):
                                    private_pem=False)
     self.assertTrue(securesystemslib.formats.PEMRSA_SCHEMA.matches(public_pem))
 
+    # Test encrypted private pem
+    private_pem_encrypted = KEYS.create_rsa_encrypted_pem(private_pem, "pw")
+    private_pem_encrypted = KEYS.extract_pem(private_pem_encrypted,
+                                   private_pem=True)
```

Comment:
I know that there are still many instances of this style of indentation in the codebase, but we should try to use the recently added indentation rules of the code style guidelines from now on.

Here is what the guidelines says about indentation for this particular case:

> Make sure to indent the continued line appropriately; double-indentation (4 spaces) is prefered for line continuation, to distinguish it from functional indentation.

Correct me if I'm wrong, but this line should look as follows:
```
KEYS.extract_pem(private_pem_encrypted,
    private_pem=True)
```